Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -190,7 +190,7 @@ static float std_dev_points(float z_pt[NPP + 1], const bool _0p_cal, const bool
  */
 static float calibration_probe(const xy_pos_t &xy, const bool stow) {
   #if HAS_BED_PROBE
-    return probe_at_point(xy, stow ? PROBE_PT_STOW : PROBE_PT_RAISE, 0, false);
+    return probe_at_point(xy, stow ? PROBE_PT_STOW : PROBE_PT_RAISE, 0, true);
```

Comment:
There may be more `calibration_probe` work needed, because in some instances the way calibration works is to put the probe at some position and then look at the tower positions. If the probe has any offset at all, reading the tower positions and trying to make sense of them is somewhat pointless. Or at least, much more complex.